src/NuGet.Core/NuGet.CommandLine.XPlat/PackCommand.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using NuGet.Commands;
+using NuGet.Logging;
+
+namespace NuGet.CommandLine.XPlat
+{
+    class PackCommand : Command
+    {
+        public PackCommand(CommandLineApplication app, Func<ILogger> getLogger)
+        {
+            app.Command("pack", pack =>
+            {
+                pack.Description = Strings.PackCommand_Description;
+
+                var basePath = pack.Option(
+                    "-b|--basePath <basePath>",
```

Comment:
We need to decide if xplat flags should use pascal casing or separate with hyphens. I know this matches nuget.exe, but other xplat commands are different right now.